.travis.yml

```diff
@@ -0,0 +1,60 @@
+language: scala
+
+sudo: required
+
+dist: trusty
+
+# Install Docker on Travis.  Merely exposing Docker through the Travis
+# settings is insufficient, because the docker-compose tests we use are
+# unable to see it in that case.
+install:
+  - curl -sSL "https://get.docker.com/gpg" | sudo -E apt-key add -
+  - echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" | sudo tee -a /etc/apt/sources.list
+  - sudo apt-get update
+  - sudo apt-get -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --assume-yes install docker-engine
+  - docker version
+
+cache:
+  directories:
+    - "~/.sbt"
+    - "~/.ivy2"
+    - "project/target/resolution-cache"
+    - "project/target/streams"
+    - "api/target"
+    - "calm_adapter/target"
+    - "common/target"
+    - "id_minter/target"
+    - "ingestor/target"
+    - "transformer/target"
+
+matrix:
+  include:
+    - scala: 2.12.2
+      env: PROJECT=common
+    - scala: 2.12.2
+      env: PROJECT=api
+    - scala: 2.12.2
+      env: PROJECT=transformer
+    - scala: 2.12.2
+      env: PROJECT=calm_adapter
+    - scala: 2.12.2
+      env: PROJECT=ingestor
+    - scala: 2.12.2
+      env: PROJECT=id_minter
```

Comment:
\>_>

It seems to be inconsistent between local and remote:

```console
$ sbt about
[info] Set current project to terraform (in build file:/Users/chana/repos/platform-infra/terraform/)
[info] This is sbt 0.13.13
[info] The current project is {file:/Users/chana/repos/platform-infra/terraform/}terraform 0.1-SNAPSHOT
[info] The current project is built against Scala 2.10.6
```

From the Circle logs:

```
Resolving org.scala-lang#scala-library;2.11.8 ...
```

2.12.2 is definitely wrong (I’m not sure where I got that from), but we should look to see if we can make set consistent.